### DISCUSSION TITLE
Resuming work on Annotation after saving and closing 3DSlicer

### DISCUSSION THREAD

**Discussion by BK0x7C8** [Project-MONAI/MONAILabel#1017] [Q&A]
Dear Folks,

I am facing an issue right now. Last week, I worked on annotating a dataset and submitted the label onto the MONAI label server. Today, I’d like to resume working on it. But when I load it into slicer (Open dataset, load the segmentation from final-directory -> Upload to MONAI Label server), the label is visible but not connected to the corresponding label in segmentation editor.

My question is, how can I resume working on a label? If I would submit that label, the file in final-directory does not show it anymore (since it was not linked, I guess)

Thanks in advance and have a great week!

**Comment by tangy5** [comment]
Hi  @r00tdotexe , thanks for the question. MONAI Label server will consider files that have labels in "final" folder as completed, thus skipping loading it. For the resume, it's needed to let MONAI Label server know it's not "complete" yet. 
Based on your description, I guess a potential solution is:
1. First move "half-done" label out from "final" directory, to "original" or another folder. (if the label is in final directory)
2. Then start MONAI Label server, Slicer to load and find that image.
3. Under MONAI Label plugin interface at Slicer, go to Tools panel (below SmartEdit and Scribbles), select import label the file to open the "half-done" label, and click the button on the right to load the current label.

The image and label can then be resumed, Segment Editor should work for adding/adjusting labels. 

Hope this helps, thanks again for using MONAI Label.

**Reply by BK0x7C8** [reply]
Hey @tangy5,

thanks alot for your reply! It works perfectly!

Cheers from Hamburg,
Bjørn
